staker/_TEST_staker_collect_reward_test.gno

```diff
@@ -158,8 +158,8 @@ func TestUnstakeToken(t *testing.T) {
 		shouldEQ(t, gnft.OwnerOf(tid(2)), test1)
 
 		// check reward
-		shouldEQ(t, gns.BalanceOf(a2u(test1)), 100000000325)    // internal
```

Comment:
Why is this number has been changed?

Reply:
due to bit of extra function call in new test case, staking duration is changed and therefore balance is litte differ